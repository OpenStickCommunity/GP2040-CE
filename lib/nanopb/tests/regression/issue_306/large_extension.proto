syntax = "proto2";

message Foo {
  extensions 1 to max;
}

extend Foo {
  optional int32 foo_ext = 99999;
}

