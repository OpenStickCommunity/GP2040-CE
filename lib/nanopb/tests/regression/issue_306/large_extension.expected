PB_BIND\(foo_ext_extmsg, foo_ext_extmsg, 4\)
